app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,72 @@
+class TweetsController < ApplicationController
+ 
+ get '/tweets' do #shows all tweets
+  if logged_in?
+   @tweets = Tweet.all
+   erb :'/tweets/tweets'
+  else
+   redirect to '/login'
+  end
+ end
+ 
+  get '/tweets/new' do #create a new tweet
+   if logged_in?
+    erb :'/tweets/create_tweet'
+   else
+    redirect to '/login'
+   end
+  end
+  
+  post '/tweets' do #create a new tweet
+   if !params[:content].empty?
+    @tweet = Tweet.create(:content => params[:content], :user_id => session[:user_id])
```

Comment:
Another way to do this is to make use of the `has_many` relationship on User:
```
@tweet = current_user.tweets.create(content: params[:content])
```
Both approaches are good, but in some cases you may find this option useful.